src/forward/lib.jl

```diff
@@ -62,5 +65,15 @@ _pushforward(dargs, ::typeof(getproperty), x, f) =
 @tangent literal_getproperty(t, ::Val{i}) where i =
   getproperty(t, i), (ṫ, _) -> getproperty(ṫ, i)
 
+@tangent literal_getproperty(t::DataType, ::Val{i}) where i =
```

Comment:
Types should never have gradients, so this can just return a zero.

Reply:
Gotcha, that would be indeed right!